### DISCUSSION TITLE
Practica intro

### DISCUSSION THREAD

**Discussion by AdanMarce** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#99] [General]
No entendí que se debe de hacer se tiene que hacer un HTML para el abogabot oh tenemos que contestar los pdf que mandaron xd ?

**Comment by AngelCruzO** [comment]
Tenemos que construir los archivos, ve las grabaciones de la semana 1. Para HTML no se ha dejado nada

**Comment by SandroBorrallez** [comment]
Tienes que hacer toda la parte de documentación, eso es lo que comprende las practicas que dejó el MC en la INTRO, y lo que respecta a HTML aun no se tiene que hacer nada para la aplicación, ya que esta semana apenas estamos empezando con el tema. Saludos!!

**Reply by AdanMarce** [reply]
Pensé que teníamos que hacer algo en HTML xd pero pues estoy en ceros con el

**Comment by roge64** [comment]
Si te sirve de ejemplo. dale una vista a mi repositorio. No es perfecto pero creo que te puede ayudar.

https://github.com/roge64/Practicas-de-Intro-a-Frontend

Saludos!

**Reply by AdanMarce** [reply]
Graciaaas pana

**Reply by RonnieRuuz** [reply]
Me sirve como referencia, gracias!
